### PR TITLE
Create comment

### PR DESCRIPTION
**Pull request by pmaslana**

### CODE REVIEW

File: internal/github/checkUnsigned.go [hunk 1]
```diff
@@ -96,3 +99,33 @@ func hasUnsignedCommits(ctx context.Context, githubClient *github.Client, pr *gi
...
+			slogs.Logr.Info("Comment already exists", "repo", repo, "PR", prNumber)
+			return nil
+		}
+	}
+
+	// Post the comment if it doesn't exist
+	comment := &github.IssueComment{
+		Body: github.String(commentBody),
+	}
+	slogs.Logr.Info("Comment", "comment", comment)
```
Comment (Starttoaster) [inline internal/github/checkUnsigned.go:124]:
```suggestion
	slogs.Logr.Info("Adding unsigned commit comment", "owner", owner, "repo", repo, "PR", prNumber, "comment", comment)
```
Adds context to the log

Comment (Starttoaster) [reply internal/github/checkUnsigned.go:124]: I'm not sure the comment really needs to be a part of it either. If you know the version of the app, you can look up in the code what the comment would have been (or in the PR timeline.) But I left that in my suggestion.

File: internal/github/checkUnsigned.go [hunk 2]
```diff
@@ -96,3 +99,33 @@ func hasUnsignedCommits(ctx context.Context, githubClient *github.Client, pr *gi
...
+	commentBody := "Your commits are not signed and our branch protection rules require signed commits. For more information on how to create signed commits, please visit this page: https://docs.chia.net/contribution/using-github/#setup-commit-signing. Please use the button towards the bottom of the page to close this pull request and open a new one with signed commits."
+	comments, _, err := client.Issues.ListComments(ctx, owner, repo, prNumber, nil)
+	if err != nil {
+		return fmt.Errorf("error fetching comments: %v", err)
+	}
+
+	// Check if the comment already exists
+	for _, comment := range comments {
+		if comment.GetUser().GetLogin() == commentAuthor && strings.EqualFold(comment.GetBody(), commentBody) {
+			slogs.Logr.Info("Comment already exists", "repo", repo, "PR", prNumber)
```
Comment (Starttoaster) [inline internal/github/checkUnsigned.go:115]:
```suggestion
			slogs.Logr.Info("Unsigned commit comment already exists", "repo", repo, "PR", prNumber)
```

Comment (pmaslana) [reply internal/github/checkUnsigned.go:115]: Incorporated the change

File: internal/github/checkUnsigned.go (hunk 1 above)
Comment (pmaslana) [reply internal/github/checkUnsigned.go:124]: I am deleting this. This was for testing at one point actually.

**Review by Starttoaster — APPROVED** [review]

**Pull request merged**